kolibri/core/assets/src/api_resource.js

```diff
@@ -254,7 +411,6 @@ class Resource {
   getModel(id, data = {}) {
```

Comment:
As the functionality for adding a new model is now under `createModel`, we should remove the `data` parameter, and just call `createModel`.